dmriprep/interfaces/images.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Image tools interfaces
+~~~~~~~~~~~~~~~~~~~~~~
+
+"""
+
+from nipype import logging
+from nipype.interfaces.base import (
+    TraitedSpec, BaseInterfaceInputSpec, SimpleInterface, File
+)
+
+LOGGER = logging.getLogger('nipype.interface')
+
+
+class MeanB0InputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='dwi file')
+    in_bval = File(exists=True, mandatory=True,
+                   desc='File containing the b-values for all volumes in the dwi series')
+    in_bvec = File(exists=True, mandatory=True,
+                   desc='File containing the b-vectors for all volumes in the dwi series')
+
+
+class MeanB0OutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='mean b0 file')
+
+
+class MeanB0(SimpleInterface):
+    input_spec = MeanB0InputSpec
+    output_spec = MeanB0OutputSpec
+
+    def _run_interface(self, runtime):
+        self._results['out_file'] = mean_b0(
+            self.inputs.in_file,
+            self.inputs.in_bval,
+            self.inputs.in_bvec,
+            newpath=runtime.cwd)
+        return runtime
+
+
+def mean_b0(in_file, in_bval, in_bvec, newpath=None):
+    """Averages the *b0* volumes from a dwi dataset"""
+    import numpy as np
+    import nibabel as nib
+    from dipy.io import read_bvals_bvecs
+    from dipy.core.gradients import gradient_table
+    from nipype.utils.filemanip import fname_presuffix
+
+    out_file = fname_presuffix(
+        in_file, suffix='_mean_b0', newpath=newpath)
+
+    img = nib.load(in_file)
+    data = img.get_data()
+
+    bvals, bvecs = read_bvals_bvecs(in_bval, in_bvec)
+    gtab = gradient_table(bvals, bvecs)
+
+    b0s = data[:, :, :, gtab.b0s_mask]
```

Comment:
something along these lines?
```suggestion
bvals = np.loadtxt(in_rasb, usecols=-1, skiprows=1)
```